nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -114,22 +118,67 @@ def get_vocab():
         self._wer = WERBPE(tokenizer=self.tokenizer, batch_dim_index=0, use_cer=False, ctc_decode=True)
 
     def transcribe(self, paths2audio_files: List[str], batch_size: int = 4) -> List[str]:
```

Comment:
This looks awfully similar to EncDecCTCModel.transcribe(...)  My understanding that the only difference is the type f self._wer (which is WERBPE in this case).   Can we move this method to ASRModel then?

Reply:
Entire the same, but its signature to the dataloader setup is different (doesnt have labels, uses tokenizer instead). Any idea about how that should be refactored? Maybe a method which does the general steps but takes a dictionary topass to the dataloader?

Reply:
Ive tried adding an internal (semi private) method - `def _setup_transcribe_dataloader(self, config: Dict) -> DataLoader:` which can be overriden by subclass.